tests/minifold/test_minifold.py

```
@@ -0,0 +1,175 @@
+
+    # Test sequence list
+    seq_list = [test_sequences["short"], test_sequences["medium"]]
+    validated = core._validate_sequences(seq_list)
+    assert isinstance(validated, list), "Should return a list"
+    assert len(validated) == 2, "Should contain two sequences"
+    assert validated == seq_list, "Sequences should be unchanged"
+
+    # Test invalid sequence
+    with pytest.raises(ValueError) as exc_info:
+        core._validate_sequences(test_sequences["invalid"])
+    assert "Invalid amino acid" in str(exc_info.value), f"Expected 'Invalid amino acid', got {str(exc_info.value)}"
```

Ruff (0.15.1)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)